drivers/led/Kconfig.pwm

```diff
@@ -3,6 +3,7 @@
 
 config LED_PWM
 	bool "PWM LED driver"
-	depends on PWM
+	default y
+	depends on PWM && DT_HAS_PWM_LEDS_ENABLED
```

Comment:
Same goes for the PWM LEDs. They can either be controlled directly via the PWM API or via the dedicated PWM LED driver.

Reply:
Not sure I follow what you are suggestion we change here?

Reply:
Ditto.